core/src/test/resources/seed_mini.sql

```diff
@@ -207,6 +213,19 @@ INSERT INTO "gene_alias" ("ENTREZ_GENE_ID","GENE_ALIAS") VALUES (3845,'KRAS2');
 INSERT INTO "gene_alias" ("ENTREZ_GENE_ID","GENE_ALIAS") VALUES (4893,'N-ras');
 INSERT INTO "gene_alias" ("ENTREZ_GENE_ID","GENE_ALIAS") VALUES (4893,'NCMS');
 
+INSERT INTO `reference_genome` VALUES (1, 'human', 'hg19', 'GRCh37', NULL, 'http://hgdownload.cse.ucsc.edu/goldenPath/hg19/bigZips/hg19.chrom.sizes', '2009-02-01 00:00:00');
```

Comment:
I'm not sure what the URL is suppose to point to here - but is the more general download page more appropriate?

[http://hgdownload.cse.ucsc.edu/goldenPath/hg19/bigZips/](http://hgdownload.cse.ucsc.edu/goldenPath/hg19/bigZips/)

Reply:
you are right, it would be better, changed.